### PR TITLE
LPD-37372 LayoutPermissionsUpgradeProcess runs twice when upgrading to 7.4 from 7.2

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPD-37372

**Motivation**

Prevent the LayoutPermissionsUpgradeProcess from running twice. 

This upgrade process was introduced in 7.1.0-a1 and moved to a different module in 7.3.0-ga1. As a result, databases upgrading from intermediate versions run the process again without making any changes, since their data was already adjusted in the first run. For more details, see: https://liferay.atlassian.net/browse/LPD-37372?focusedCommentId=2750679

**Proposed Approach**

Check within the LayoutPermissionsUpgradeProcess if it has already been executed previously using the servletContextName of the previous module. If so, skip the process logic.

@achaparro @javalosjr96 I think this solution addresses the issue. Could you please provide feedback on this approach? Thank you very much!

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:upgrade

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
@lfbesada I've executed relevant, and I'm going to put it on hold waiting for the review of Database Infra Team

**Comment by achaparro** [comment]
ci:test:upgrade

File: modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_0_0/LayoutPermissionsUpgradeProcess.java [hunk 1]
```diff
@@ -68,4 +73,29 @@ protected void doUpgrade() throws Exception {
 		}
 	}
 
+	private boolean _skipUpgrade() throws Exception {
```
Comment (achaparro) [inline modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_0_0/LayoutPermissionsUpgradeProcess.java:76]: @lfbesada is there any reason you can not use the existing parent method com.liferay.portal.kernel.upgrade.UpgradeProcess#isSkipUpgradeProcess? If possible, please, use it.

Thanks.

**Comment by lfbesada** [comment]
ci:report:35567397

File: modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_0_0/LayoutPermissionsUpgradeProcess.java (hunk 1 above)
Comment (lfbesada) [reply modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_0_0/LayoutPermissionsUpgradeProcess.java]: @achaparro I didn't know about that method! Thanks a lot for your review and feedback, I've added it. Regards

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:test:upgrade

File: modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_0_0/LayoutPermissionsUpgradeProcess.java (hunk 1 above)
Comment (achaparro) [reply modules/apps/layout/layout-service/src/main/java/com/liferay/layout/internal/upgrade/v1_0_0/LayoutPermissionsUpgradeProcess.java]: No problem :-) The rest of the changes look good to me from upgrade perspective.

Thanks!

**Comment by lfbesada** [comment]
ci:report:36366153

**Pull request closed without merging**